backend/model/Recipe.mjs

```diff
@@ -4,7 +4,11 @@ const { Schema, model } = mongoose;
 const schema = new Schema(
   {
     id: { type: Number },
```

Comment:
The `id` is always given by MongoDB. I would probably not set this in the Schema.